global-test.jboss-eap-xp.openshift.properties

```diff
@@ -7,12 +7,12 @@
 ###############################################
 
 # Databases
-intersmash.mysql.image=quay.io/centos7/mysql-80-centos7:centos7
-intersmash.postgresql.image=quay.io/centos7/postgresql-13-centos7:centos7
+intersmash.mysql.image=registry.redhat.io/rhel10/mysql-84:latest
```

Comment:
@fabiobrz is that intended? I mean switching from quay.io to registry.redhat.io ...

Reply:
Thanks for asking @tommaso-borgato. Yes, it is intended. 

These are defaults that we want to use when testing against products and assuming that the automation would access such registry.